lib/re.mli

```diff
@@ -371,6 +371,7 @@ val pp_re : Format.formatter -> re -> unit
 (** Alias for {!pp_re}. Deprecated *)
 val print_re : Format.formatter -> re -> unit
 
+val witness : t -> string
```

Comment:
I will. I need a lot more tests as well. But most importantly, should we
consider this function as "experimental" or "internal"? It's only really being
added for tyre so it can be changed/removed on any whim tyre might have.

On 09/12, Gabriel Radanne wrote:
>> @@ -371,6 +371,7 @@ val pp_re : Format.formatter -> re -> unit
>>  (** Alias for {!pp_re}. Deprecated *)
>>  val print_re : Format.formatter -> re -> unit
>>
>> +val witness : t -> string
>
>Please include the documentation.
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/ocaml/ocaml-re/pull/122/files/6fd311ebfd45376a0e7ff3f610f184e074fb5e23#r78395833